deployment/terraform/create.go

```diff
@@ -230,7 +220,19 @@ func (t *Terraform) Create(initData bool) error {
 		if err := pingServer("http://" + pingURL); err != nil {
 			return fmt.Errorf("error whiling pinging server: %w", err)
 		}
+	}
 
+	// Note: This MUST be done after app servers have been set up.
+	// Otherwise, the vacuuming command will fail because no tables would
+	// have been created by then.
+	if t.output.HasDB() {
+		// Aurora sets the default_text_search_config parameter to 'simple';
+		// a more sane default for a generic deployment is 'english'
+		if t.config.TerraformDBSettings.InstanceEngine == "aurora-postgresql" {
+			if err := t.updatePostgresSettings(extAgent); err != nil {
+				return fmt.Errorf("could not modify default_search_text_config: %w", err)
+			}
+		}
 	}
```

Comment:
We need to update the comment and the error message here, since we do more in `updatePostgresSettings` now.